editoast/src/views/timetable/stdcm.rs

```diff
@@ -152,6 +160,9 @@ pub(in crate::views) async fn stdcm(
     Query(query): Query<InfraIdQueryParam>,
     Json(request): Json<Request>,
 ) -> Result<Json<StdcmResponse>> {
+    // Add serialized request to trace attributes
+    Span::current().record("request", serde_json::to_string(&request)?);
```

Comment:
The request should already be part of the span since you added `fields(request)`.
Is it required to have it serialize in json?

Reply:
`fields(request)` just declares the field, it's not automatically filled.

What *could* have been an option instead was to not put `skip_all` in the trace and instead list all parameters as excluded, except `request`. Then it would automatically log the parameter. But it's not actually saved as json, it uses the debug string (or something similar). It's not as easily importable in a script.

Reply:
Having it attached to the span is very convenient though, it's the place we'd be looking for it. Or do you have some other kind of logging in mind? I have seen a "log" tab in the span, but I don't quite know how to attach things there